src/Client/Web/Bit.Client.Web.BlazorUI.Tests/Lists/BitBasicListTests.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using Bunit;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using static Bit.Client.Web.BlazorUI.Tests.Lists.BitBasicListTest;
+
+namespace Bit.Client.Web.BlazorUI.Tests.Lists
+{
+    [TestClass]
+    public class BitBasicListTests : BunitTestContext
+    {
+
+        [DataTestMethod, DataRow(1000)]
+        public async Task BitBasicListShoudReturnExpectedChildElements(int itemCount)
+        {
+            var component = RenderComponent<BitBasicListTest>();
+
+            var bitList = component.Find(".bit-basic-list");
+            int comItemCount = bitList.ChildElementCount;
+
+            Assert.AreEqual(comItemCount, itemCount + 1);
+        }
+        [DataTestMethod, DataRow(true)]
+        public async Task BitBasicListCheckWithVirtualize(bool vitualize)
+        {
+            var component = RenderComponent<BitBasicListTest>();
+            var bitList = component.Find(".bit-basic-list");
+
+            var height = bitList.FirstElementChild.GetAttribute("style").Split(":")[1].Replace("px;", "");
+
+            Assert.AreEqual("0", height.Trim());
+        }
+
+
+        [DataTestMethod, DataRow(false, "testItem")]
+        public async Task BitBasicListCheckWithoutVirtualize(bool vitualize, string childClass)
```

Comment:
If it's a check without "Virtualize" then there is no need to take the option from method arguments. it is false by definition in this method